packages/editor/src/plugins/image/components/image-selection-screen.tsx

```diff
@@ -30,15 +32,32 @@ export function ImageSelectionScreen(
   const imgUrl = src.get() as string
   const showErrorMessage = imgUrl.length > 5 && !isImageUrl(imgUrl)
 
+  const [showPixabayModal, setShowPixabayModal] = useState(false)
+
+  const onSelectPixabayImage = (imageUrl: string) => {
```

Comment:
tiny thing, but  `imgUrl`, `isImageUrl` and `imageUrl` following each other so closely makes me look twice.

Reply:
Changed imgUrl to imagUrl now for consistency, since  the API returns something with "largeImageUrl"